eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/BasicTypeSpec.csproj

```diff
@@ -12,5 +12,6 @@
   <ItemGroup>
     <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
     <PackageReference Include="System.Text.Json" Version="8.0.4" />
+		<PackageReference Include="Azure.Core" Version="1.43.0" />
```

Comment:
we need a format here

Reply:
hmm... this is auto-generated, and the format is correct in vscode and VS.

Reply:
It's two tabs before this line. Others are 4 spaces.

Reply:
And this is actually manually added for now, will need to generate it.